scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/core/service/impl/StorageProperties.java

```diff
@@ -0,0 +1,22 @@
+package edu.asu.diging.scriptoocloud.core.service.impl;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource("classpath:/config.properties")
+public class StorageProperties {
+
+    @Value("${rootUploadLocation}")
+    private String rootLocationString;
```

Comment:
this should be injected in the classes that need and this class should be removed